common/src/app/common/geom/shapes/common.cljc

```diff
@@ -31,6 +32,22 @@
     (gpt/point (/ (+ minx maxx) 2.0)
                (/ (+ miny maxy) 2.0))))
 
+(defn center-bounds [[a b c d]]
```

Comment:
This function receives a coll of fixed number of points, they can be directly 4 arguments?

Reply:
This will receive the `(:points shape)` value and it's a fixed value of 4 points. So we don't want to "spread" them.